test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -8089,6 +8090,49 @@ public void ToNDArray()
                 Assert.Equal(2, a.GetLength(2));
             }
         }
+        [Fact]
+        [TestOf(nameof(TorchSharp.Utils.TensorAccessor<float>.ToArray))]
+        public void ToArrayFastTensorAccessor()
+        {
+            {
+                var t = arange(0, 12, ScalarType.Float32).view(2, 3, 2).transpose(2, 1);
+                Assert.False(t.is_contiguous());
+                t = t[TensorIndex.Colon, TensorIndex.Slice(1, null)];
+                float[] v = t.data<float>().ToArray();
+                var tt = tensor(v, t.shape, t.dtype);
+                Assert.True(tt.equal(t).all().item<bool>());
+            }
+            {
+                var t = randn(4, 4, 3);
+                t = t[2, TensorIndex.Ellipsis];
+                Assert.True(t.is_contiguous());
+                t = t.transpose(1, 0);
+                Assert.False(t.is_contiguous());
+                Assert.Equal(24, t.storage_offset());
+                float[] v = t.data<float>().ToArray();
+                var tt = tensor(v, t.shape, t.dtype);
+                Assert.True(tt.equal(t).all().item<bool>());
+            }
+            {
+                var t = arange(0,128).reshape(4,4,-1);
+                t = t[2, TensorIndex.Ellipsis];
+                Assert.True(t.is_contiguous());
+                t = t[TensorIndex.Colon, 2];
+                Assert.False(t.is_contiguous());
+                Assert.Equal(66, t.storage_offset());
+                long[] v = t.data<long>().ToArray();
+                var tt = tensor(v, t.shape, t.dtype);
+                Assert.True(tt.equal(t).all().item<bool>());
+            }
+            {
+                var t = arange(0, 128).reshape(4, 4, -1);
+                Assert.True(t.is_contiguous());
+                long[] v = t.data<long>().ToArray();
+                var tt = tensor(v, t.shape, t.dtype);
+                Assert.Equal(0, t.storage_offset());
+                Assert.True(tt.equal(t).all().item<bool>());
+            }
```

Comment:
I need add more test like for example `ToArray(18, 4)` that is from index 18 with length 4. Or add `ToArray(5)` mean  from index 5 to the numel.